src/xia2/cli/cluster_analysis.py

```diff
@@ -80,8 +72,6 @@
 output {
   log = xia2.cluster_analysis.log
     .type = str
-  json = xia2.cluster_analysis.json
```

Comment:
This shouldn't be removed or the program crashes?

Reply:
This line is back in, and all scripts now calls from dials.algorithms.correlation.analysis.working_scope rather than dials.command_line.correlation.phil_scope